angular.json

```diff
@@ -101,6 +101,38 @@
       "sourceRoot": "projects/angular/src",
       "prefix": "lib",
       "architect": {
+        "storybook": {
```

Comment:
nit: I think these should be after `build` and `test`.

Reply:
And I would rename this one to `start-storybook` for clarity.

Reply:
Renamed to `start-storybook` and moved it below `build` & `test`